src/react/utils.ts

```diff
@@ -1,3 +1,4 @@
+export { HydrateAtoms } from './utils/HydrateAtoms.tsx'
```

Comment:
I didn't export the types from `src/react/utils/typeUtils.ts` to be used externally, but if we think some some be I would suggest they be:
```ts
export {
  AnyWritableAtom,
  AtomMap,
  AtomTuple,
  InferAtoms,
} from './utils/typeUtils.ts'
```